docs/internal/Level-1+-Reviews-and-Changes-Requested.md

```diff
@@ -0,0 +1,131 @@
+
+## Consolidation in disagreement (Changes-requested)
+
+- Reviewer (Level 1) - reviewing an application
+- Reviewer (Level 1) - submit ~~`LOQ (List of Questions)`~~ `Conform` or `Non Conform` advise to upper level reviewer
+- Reviewer (Level 2) - consolidate lower level review(s) and `disagree` with some advised decisions
+- Reviewer (Level 2) - submit `Changes Requested` back to reviewer (e.g. Level 1)
+- Reviewer (Level 1) - makes changes in response to `Changes Requested` and re-submit to upper level reviewer.
+
+### Schema changes
+
+![Level-1+-Review-Structure-Changes](images/Level-1+-Review-Structure-Changes.png)
+
+#### Consolidation
+
+- There is just one specific progress-bar on consolidation which considers only (one or many) lower level reviews agreement/disagreements (`consolidationProgress`).
+- On the consolidation, each element `state` has:
+  - the decision done in this current **consolidation** in `thisReviewLatestResponse`
+  - the decision done in a previous **consolidation** (if any) in `thisReviewPreviousResponse` to track changes between this and a previous consolidation (of the same user)
+- On the consolidation, each element `state` it can only have one lower level review:
+  - the `lowerLevelReviewlatestResponse` is the latest decision submitted (or unchanged) from a lower level review
+  - and `lowerLevelReviewPreviousResponse` to track changes between the latest submitted review and this one (after changes-requested to lower level reviewer)
+- **TODO**: Not really sure exact the current use of `latestOriginalReviewResponse` and `previousOriginalReviewResponse` so will wait for feedback to complete this one....
+
+#### Review
+
+- The progress for a first-review (if no changes-requested) is done only with (`reviewProgress`)
+- If some changes requested:
+  - used both `reviewProgress` and `changeRequestsProgress` to track review progress
+  - each element `state` has `isChangeRequest` and `isChanged` (if one of the change-requested) to track if changes are done
+- On the review, similarly to consolidation each element `state` has:
+  - the decision done in this current **review** in `thisReviewLatestResponse`
+  - the decision done in a previous **review** (if any) in `thisReviewPreviousResponse` to track changes between this and a previous review (of the same user)
+- To find the consolidator change-requested to this reviewer:
+  - if it is stored inside `thisReviewPreviousResponse` (when review is in DRAFT) or `thisReviewLatestResponse` (if review in CHANGES_REQUESTED hasn't started)
+
+## Submitting back to Applicant
+
+When a upper level reviewer disagrees with some decision on lower level it needs to be changed by lower level on a new review.
+
+The consolidation **needs to happen again and be in agreement** so it can be send to Applicant (with Reviewer Level 1 comments) in the form of `LOQ (List of Questions)`.
```

Comment:
```suggestion
The consolidation **needs to happen again and be in agreement** so it can be sent to Applicant (with Reviewer Level 1 comments) in the form of `LOQ (List of Questions)`.
```